indy_node/server/action_req_handler.py

```diff
@@ -30,39 +30,38 @@ def __init__(self, idrCache: IdrCache,
         self.info_tool = info_tool
         self.poolManager = poolManager
         self.poolCfg = poolCfg
+        self.write_req_validator = WriteRequestValidator(config=getConfig(),
+                                                         auth_map=authMap,
+                                                         cache=self.idrCache,
+                                                         anyone_can_write_map=anyoneCanWriteMap)
 
     def doStaticValidation(self, request: Request):
         pass
 
     def validate(self, req: Request):
-        status = None
         operation = req.operation
         typ = operation.get(TXN_TYPE)
         if typ not in self.operation_types:
             return
         origin = req.identifier
         try:
```

Comment:
Do we still need this try-except? This looks like a generic logic for checking if there is a role. It needs to be part of `write_req_validator.validate`

Reply:
Done.